src/TextUI/Help.php

```diff
@@ -70,7 +70,7 @@ final class Help
             ['arg' => '--uses <name>', 'desc' => 'Only runs tests annotated with "@uses <name>"'],
             ['arg' => '--list-tests', 'desc' => 'List available tests'],
             ['arg' => '--list-tests-xml <file>', 'desc' => 'List available tests in XML format'],
-            ['arg' => '--filter <pattern>', 'desc' => 'Filter which tests to run'],
+            ['arg' => '--tests-xml <file>', 'desc' => 'Filter which tests to run in XML format'],
```

Comment:
Does this replace the `--filter` option?

Reply:
No, I think you spotted a bug I introduced after I rebased with master which changed similarly lines my PR did.

Thanks, will fix ASAP!

Reply:
Re-instated, thank you @spawnia